examples/grafana/grafana-ocp.yaml

```diff
@@ -10,13 +10,40 @@ metadata:
     iconClass: icon-cogs
     tags: "metrics,monitoring,grafana,prometheus"
 parameters:
+- description: The location of the proxy image
+  name: IMAGE_GF
+  value: mrsiano/grafana-ocp:latest
+- description: The location of the proxy image
+  name: IMAGE_PROXY
+  value: openshift/oauth-proxy:v1.0.0
 - description: External URL for the grafana route
   name: ROUTE_URL
   value: ""
 - description: The namespace to instantiate heapster under. Defaults to 'grafana'.
   name: NAMESPACE
   value: grafana
+- description: The session secret for the proxy
+  name: SESSION_SECRET
+  generate: expression
+  from: "[a-zA-Z0-9]{43}"
 objects:
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    name: grafana-ocp
```

Comment:
Can we remove the `-ocp` suffix from everywhere? I don't see what value does it add.

Reply:
sure